### PR TITLE
HWP-204: Authenticate mongoose connection from API

### PR DESCRIPTION
**Pull request by BradyMitch**
# Description

Fixed the API's connection to the mongo database so that it connects to the correct database with correct authentication.
REQUIRES ENV FILE TO BE UPDATED

This PR includes the following proposed change(s):

- Created an admin user for the database in init-mongodb/create-db.js
- API connection in db.js uses an updated mongo URI connection string that authenticates the connection to the database

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Built the docker images, successfully registered and logged in to web app and saw data show up in the collection when using the mongo shell in Docker Desktop. Tested by @ZachBourque and @Bottle7 

## Does the change impact or break the Docker build?

- [x] Yes
- [ ] No

Changes ENV variables used by mongodb container

If Yes: Has Docker been updated accordingly?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .env-template [hunk 1]
```diff
@@ -1,4 +1,12 @@
 API_REF=
-MONGO_URI=
 API_PORT=
+
```
Comment (akroon3r) [inline .env-template:3]:
```suggestion
```

Comment (akroon3r) [reply .env-template:3]: if we need to break this down into sections for API and Frontend vars I suggest we do so with comment headers like we did in the docker compose.

File: .env-template [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
+MONGO_URI=
+MONGO_ROOT_USERNAME=
+MONGO_ROOT_PASSWORD=
+# MONGO_DATABASE is only used in docker-compose.
```
Comment (akroon3r) [inline .env-template:7]: these type of comments should be in our project README imo, under a section about environment variables. Thoughts?

File: .env-template (hunk 1 above)
Comment (BradyMitch) [reply .env-template:6]: Added headers. Is your suggestion to remove the blank line at the end of the file?

File: .env-template (hunk 2 above)
Comment (BradyMitch) [reply .env-template:13]: Agreed. Will make changes. Makes more sense that way.

**Review by BradyMitch — COMMENTED** [review]
Synced env and env-template, comment headers added.

**Review by ZachBourque — APPROVED** [review]
Looks good

**Review by Bottle7 — APPROVED** [review]

**Pull request merged**